eq-author/src/components/Answers/MultipleChoiceAnswer/__snapshots__/Option.test.js.snap

```diff
@@ -104,6 +104,7 @@ exports[`Option should render a checkbox 1`] = `
   font-weight: inherit;
```

Comment:
Why is this file changing?

Reply:
Its because I put some styling on the wrapping textbox to stop it wrapping and it has affected the this snapshot. I think maybe I need to do the styling closer to the alias textbox so it doesn't affect the multiple choice answers